src/vue-property-decorator.ts

```diff
@@ -149,6 +149,23 @@ export function Emit(event?: string): MethodDecorator {
   }
 }
 
+/**
+ * decorator of a ref prop
+ * @param refKey the ref key defined in template
+ */
+export function Ref(refKey?: string) {
+  return createDecorator((options, key) => {
+    options.computed = options.computed || {};
+    options.computed[key] = {
+      cache: false,
```

Comment:
This `cache` property is not supported I think as I read the current api reference of Vue.js.

Reply:
It's deprecated, but it is supported. And `cache: false` is necessary here because `$refs.*` is not reactive. If their value is cached, Vue won't be able to update them.

Reply:
OK, I understand. Thanks for your explanation.